tests/integration/targets/azure_rm_managementgroup/aliases

```diff
@@ -1,3 +1,2 @@
 cloud/azure
 shippable/azure/group2
-disabled
```

Comment:
@paultaiton Please Disable this test because this module has too many permissions to automate tests. Thank you very much!

Reply:
> @paultaiton Please Disable this test because this module has too many permissions to automate tests. Thank you very much!

OK, should be good now